### ISSUE TITLE
Assign ministry team field

### ISSUE THREAD

**Issue by lmullane** [bcgov/foi-flow#500]
* As a ministry coordinator or IAO analyst
* I want a field on the request view to assign a ministry team/coordinator
* so that I know who is responsible for a request at the ministry

**Assumptions & Scope**	
What are the assumptions for this story?

What is in and not in scope?

**Acceptance Criteria**	

**Scenario 1: Call for Records**
* GIVEN a request is in the Call for Records and onwards state
* WHEN the user is on the request view
* THEN the user will see the assigned ministry team
https://jacklyn808742.invisionapp.com/console/share/3W2O64AP5C/768088423

**Scenario 2: IAO analsyt**
* GIVEN ... (you or your condition)
* WHEN ... (what you do) 
* THEN ... (what you see) 

**Scenario 3: xxxxxx**
...

**Dependencies? What is the impact of this dependency? (If so, link dependency in the ticket, make it visible in a team´s backlog)**

**Validation Rules? (If yes, list here)**	

**Design**
@xxx - please link the Design here

**Definition of Ready**

1. [ ] Is there a well articulated User Story?
2. [ ] Is there Acceptance Criteria that covers all scenarios (happy/sad paths)?
3. [ ] If there is a user interface, is there a design? 
4. [ ] Does the user story need user research/validation?
5. [ ] Does this User Story needs stakeholder approval?
6. [ ] Design / Solution accepted by Product Owner
7. [ ] Is this user story small enough to be completed in a Sprint? Should it be split?
8. [ ] Are the dependencies known/ understood? (technical, business, regulatory/policy)
9. [ ] Has the story been estimated?

**Definition of Done**

1.    [ ] Passes developer unit tests
2.    [ ] Passes peer code review
3.    [ ] If there's a user interface, passes UX assurance
4.    [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
5.	[ ] Confirm Test cases built and succeeding 
6.	[ ] No regression test failures
7.	[ ] Test coverage acceptable by Product Owner
8.	[ ] Ticket ready to be merged to master or story branch
9.	[ ] Developer to list Config changes/ Update documents and designs
10.	[ ] Can be demoed in Sprint Review 
11.	[ ] Tagged as part of a Release
12.   [ ] Feature flagged if required
13.  [ ] Change Management activities done?

**Comment by lmullane** [comment]
@arielleandrews , @JHarrietha-AOT, please read the scenarios above.

Question - should the IAO analyst be able to change the ministry assignee? I'm thinking no. They assign it to the team when they change the state to CFR. That way a request is not missed.

The only person who needs to be able to change who the ministry assigned to field is the ministry coordinator.

How do you think it should work?

cc @abin-aot

**Comment by arielleandrews** [comment]
Agree @lmullane, only IAO staff should be able to change the IAO assignee, and only Ministry staff should be able to change the Ministry assignee. I don't think we need to restrict which Ministry staff can change their own assignees at this point. If the Coordinator is away their Manager or back-up person may want to change the assignee

**Comment by JHarrietha-AOT** [comment]
@arielleandrews That's what I was thinking as well Ministries and IAO want to have the reference of the other who is assigned, but they only need to edit their respective fields. Otherwise looks good.

**Comment by lmullane** [comment]
Agreed. Anyone on the ministry team can change the ministry assignee (for the pilot in our system there will be only 1 role which is ministry coordinator). We may add future roles when we release reporting functionality (i.e. reviewers: ADMS, DMs)

Appreciate how we are thinking through these requirements as a team!

**Comment by arielleandrews** [comment]
Thank you @divyav-aot and @richard-aot for clarifying.

QA completed, all scenarios passed in Chrome, Edge and Firefox
